cdap-master/src/main/java/co/cask/cdap/master/startup/ConfigurationCheck.java

```diff
@@ -100,6 +101,15 @@ private void checkServiceResources(Set<String> problemKeys) {
     }
   }
 
+  private void checkBindAddress() {
+    // check if app.bind.address is 127.0.0.1 or localhost
+    String value = cConf.get(Constants.AppFabric.SERVER_ADDRESS);
+    if ("127.0.0.1".equals(value) || "localhost".equals(value)) {
+      LOG.warn(Constants.AppFabric.SERVER_ADDRESS + " is set to " + value + ". This is invalid in any multinode " +
```

Comment:
In the error message, should include the name of the configuration (i.e. `Constants.AppFabric.SERVER_ADDRESS`)